config/packages/monolog.yaml

```diff
@@ -48,8 +48,8 @@ when@prod:
                 type: fingers_crossed
                 action_level: error
                 handler: nested
-                buffer_size: 50 # How many messages should be saved? Prevent memory leaks
                 excluded_http_codes: [404, 405]
+                buffer_size: 50 # How many messages should be saved? Prevent memory leaks
```

Comment:
I just moved these lines around, so it's the same as a Symfony from scratch. Nothing special here.